src/loaders/genome_attributes/parse_computed_genome_attribs.py

```diff
@@ -0,0 +1,247 @@
+"""
+PROTOTYPE
+
+This script involves processing computed genome attributes and organizing them into a structured format suitable
+for importing into ArangoDB. The resulting JSON file will be used to update (upsert/insert) the database with the parsed
```

Comment:
Again, I don't think upserting should ever happen, right? It's either insert or overwrite

Reply:
👍 updated doc

Reply:
:+1: